### DISCUSSION TITLE
technology/works/theme

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#105] [Announcements]
# technology/works/theme

简约风的vitepress博客主题介绍

https://sugarat.top/technology/works/theme.html

**Comment by guowei-gong** [comment]
主页风格看起来不错，但是详情的左侧这种目录我不太感冒~

**Reply by ATQQ** [reply]
没找到好的设计 大兄弟有看到比较好的吗 推荐一下

**Comment by ATQQ** [comment]
当然如果不需要 这部分内容 可以 通过设置 隐藏 https://theme.sugarat.top/config/global.html#recommend
```ts
const blogTheme = getThemeConfig({
  recommend: false
})
```

**Comment by MuYi086** [comment]
配置的本地logo，dev能正常显示，build后404找不到logo文件, 问题出在哪呢？
```pre
home: {
    name: "Blog",
    motto: '记录工作和生活的点滴',
    logo: '/Images/home/logo.png'
  }
```

**Reply by ATQQ** [reply]
这里填写的地址构建时不会被替换。你将这个images移动到public目录下 即可
